packages/walletconnect/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@celo/wallet-walletconnect",
-  "version": "4.1.0",
+  "version": "4.2.1",
```

Comment:
this might need to be a major change since walletConnectId is now required

Reply:
I don't actually know it's really required

Reply:
nvm it is required according the the docs